build-system/tasks/pr-deploy-bot-utils.js

```diff
@@ -70,13 +70,17 @@ async function replaceUrls(dir) {
 }
 
 async function signalPrDeployUpload(result) {
+  // TODO(rsimha): Remove this check once Travis is shut down.
+  if (!isCircleciBuild()) {
+    return;
+  }
```

Comment:
What's this for? This will prevent the bot from working while we're still on Travis.

Reply:
Builds are already running on CircleCI. This is to prevent duplicate statuses from Travis and CircleCI.

Reply:
I see.. I think it's safer for the bot to use Travis for everything until CircleCI is ready to be switched to, instead depending on both.

Reply:
Done.